pom.xml

```diff
@@ -43,7 +43,7 @@
 
   <properties>
     <revision>3.0.0-beta6</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <changelist>-rc1791.3a94dfffd1fc</changelist>
```

Comment:
Please explain why this change is needed?  What is the benefit of modifying that changelist value?

Reply:
Well, in a previous conversation was decided not to use SNAPSHOTS

Reply:
I think you may have misunderstood the earlier conversation.  The git client plugin does not need to have its changelist value modified.  The incrementals process provided by the parent pom delivers artifacts to the incrementals repository automatically with each build of a branch or of a pull request.  

When another plugin needs an unreleased version of a dependency (like the git plugin will need the git client plugin), then the plugin which has the dependency (the git plugin) can refer to the dependency generated by incrementals (the git client plugin build).  

There should be no need to modify the changelist value.